tests/models/opt/test_modeling_opt.py

```diff
@@ -321,7 +305,7 @@ def test_logits(self):
                 [6.4783, -1.9913, -10.7926, -2.3336, 1.5092, -0.9974, -6.8213, 1.3477, 1.3477],
             ]
         )
-        assert torch.allclose(logits, logits_meta, atol=1e-4)
```

Comment:
maybe revert this since it will probs lead to merge conflicts

Reply:
IMO it makes more sense to have the self.assertTrue rather that the assert. What do you think ? I should have merge from the main on this one

Reply:
Think it doesn't matter too much tbh (probs not worth spending too much time on it :-) )